### PR TITLE
[all hosts] (Auth) Add support table for NAA preview

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR adds a table explains the current support story for NAA in Office Add-ins across WXPO.

### CODE REVIEW

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 1]
```diff
@@ -208,7 +208,14 @@ Nested app authentication enables SSO for applications that are nested inside of
 
 ## NAA supported accounts and hosts
 
-NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn’t support B2C scenarios. For preview, NAA is supported in Office on Windows and on Mac. For GA, NAA will also support Office on the web, and Outlook mobile on Android and on iOS.
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn’t support B2C scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
```
Comment (Rick-Kirkham) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:211]: What's "B2C"?

**Review by Rick-Kirkham — APPROVED** [review]
Approving with a suggestion.

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 2]
```diff
@@ -208,7 +208,14 @@ Nested app authentication enables SSO for applications that are nested inside of
...
+
+| Application | Windows                                                               | Mac | Web          | iOS | Android |
+|-------------|-----------------------------------------------------------------------|-----|--------------|-----|---------|
+| Excel       | Yes (Insiders Slow)                                                   | Yes | Coming soon  | Yes | No      |
+| Outlook     | Yes (Insiders Slow for classic Outlook only, new Outlook coming soon) | Yes | Yes          | Yes | Yes     |
+| PowerPoint  | Yes (Insiders Slow)                                                   | Yes | Coming soon  | Yes | No      |
+| Word        | Yes (Insiders Slow)                                                   | Yes | Coming soon  | Yes | No      |
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:218]:
```suggestion
| Excel       | Yes (Insiders Slow)                                                   | Yes | Coming soon  | Yes | Not applicable      |
| Outlook     | Yes (Insiders Slow for classic Outlook only, new Outlook coming soon) | Yes | Yes          | Yes | Yes     |
| PowerPoint  | Yes (Insiders Slow)                                                   | Yes | Coming soon  | Yes | Not applicable      |
| Word        | Yes (Insiders Slow)                                                   | Yes | Coming soon  | Yes | Not applicable      |
```

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 3]
```diff
@@ -208,7 +208,14 @@ Nested app authentication enables SSO for applications that are nested inside of
 
 ## NAA supported accounts and hosts
 
-NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn’t support B2C scenarios. For preview, NAA is supported in Office on Windows and on Mac. For GA, NAA will also support Office on the web, and Outlook mobile on Android and on iOS.
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn’t support B2C scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
+
+| Application | Windows                                                               | Mac | Web          | iOS | Android |
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:213]: Consider indicating iPad vs iOS support (e.g., "iPad" or "Yes (iPad)" for WXP, "iOS" or "Yes (iOS)" for Outlook)

```suggestion
| Application | Windows                                                               | Mac | Web          | iOS/iPad | Android |
```

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:211]:
```suggestion
NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support B2C scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
```

**Review by samantharamon — APPROVED** [review]

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 4]
```diff
@@ -208,7 +208,14 @@ Nested app authentication enables SSO for applications that are nested inside of
...
+|-------------|-----------------------------------------------------------------------|-----|--------------|-----|---------|
+| Excel       | Yes (Insiders Slow)                                                   | Yes | Coming soon  | Yes | No      |
```
Comment (codexeon) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:215]: new name for this channel is Current Channel (Preview)

**Review by codexeon — APPROVED** [review]

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md (hunk 4 above)
Comment (AlexJerabek) [reply docs/develop/enable-nested-app-authentication-in-your-add-in.md]: Thanks! What does Insiders Fast map to? "Beta Channel"?

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 5]
```diff
@@ -15,14 +15,20 @@ You can use the MSAL.js library (version 3.11 and later) with nested app authent
...
 > [!IMPORTANT]
 > Nested app authentication (NAA) is currently in preview. To try this feature, join the Microsoft 365 Insider Program (https://insider.microsoft365.com/join) and choose the Beta Channel. Don't use NAA in production add-ins. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
-> NAA is supported in the following builds.
->
-> - Word, Excel, PowerPoint, and Outlook on Windows build 16.0.17531.20000 or later
-> - Word, Excel, PowerPoint, and Outlook on Mac build 16.85.24040319 or later
->
-> Support for NAA in event-based activation code in Outlook is coming soon.
+
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn’t support Azure AD B2C for business-to-consumer identity management scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:23]: Isn't Azure AD a deprecated term?

Comment (AlexJerabek) [reply docs/develop/enable-nested-app-authentication-in-your-add-in.md:23]: Doesn't seem that way.
https://learn.microsoft.com/en-us/azure/active-directory-b2c/overview

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 6]
```diff
@@ -15,14 +15,20 @@ You can use the MSAL.js library (version 3.11 and later) with nested app authent
...
 > [!IMPORTANT]
 > Nested app authentication (NAA) is currently in preview. To try this feature, join the Microsoft 365 Insider Program (https://insider.microsoft365.com/join) and choose the Beta Channel. Don't use NAA in production add-ins. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
-> NAA is supported in the following builds.
->
-> - Word, Excel, PowerPoint, and classic Outlook on Windows Build 16.0.17531.20000 or later
-> - Word, Excel, PowerPoint, and Outlook on Mac Build 16.85.24040319 or later
->
-> Support for NAA in event-based activation code in Outlook is coming soon.
+
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support Azure AD B2C for business-to-consumer identity management scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:23]:
```suggestion
NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support Azure Active Directory B2C for business-to-consumer identity management scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
```

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/develop/enable-nested-app-authentication-in-your-add-in.md]: Changed to "It doesn't support Azure AD B2C for business-to-consumer identity management scenarios."

**Comment by AlexJerabek** [comment]
Thanks for the feedback @Rick-Kirkham, @ElizabethSamuel-MSFT, @samantharamon, and @codexeon! I _think_ I've addressed everything, assuming "Beta Channel" maps to what I previously understood to be called "Insiders Fast".

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 7]
```diff
@@ -15,14 +15,21 @@ You can use the MSAL.js library (version 3.11 and later) with nested app authent
...
-> - Word, Excel, PowerPoint, and Outlook on Mac Build 16.85.24040319 or later
->
-> Support for NAA in event-based activation code in Outlook is coming soon.
+
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support Azure Active Directory B2C for business-to-consumer identity management scenarios. The following table explains the current support in preview. Anything listed as "Coming soon" will be supported by the time that NAA is made generally available.
+
+| Application | Windows                                                                            | Mac | Web          | iOS/iPad   | Android        |
+|-------------|------------------------------------------------------------------------------------|-----|--------------|------------|----------------|
+| Excel       | Yes (Current Channel (Preview))                                                    | Yes | Coming soon  | Yes (iPad) | Not applicable |
+| Outlook     | Yes* (Current Channel (Preview) for classic Outlook only, new Outlook coming soon) | Yes | Yes          | Yes (iOS)  | Not applicable |
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:28]: Originally, Android was "Yes" on Outlook. Just checking if it should still say that.

```suggestion
| Outlook     | Yes* (Current Channel (Preview) for classic Outlook only, new Outlook coming soon) | Yes | Yes          | Yes (iOS)  | Yes            |
```

File: docs/develop/enable-nested-app-authentication-in-your-add-in.md [hunk 8]
```diff
@@ -15,14 +15,21 @@ You can use the MSAL.js library (version 3.11 and later) with nested app authent
 - You can use incremental and dynamic consent for scopes.
 - You don't need to [preauthorize your hosts](/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?tabs=manifest-teams-toolkit#update-azure-ad-app-registration-for-sso) (For example, Teams, Office) to call your endpoints.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/enable-nested-app-authentication-in-your-add-in.md:16]:
```suggestion
- You don't need to [preauthorize your hosts](/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?tabs=manifest-teams-toolkit#update-azure-ad-app-registration-for-sso) (for example, Teams, Office) to call your endpoints.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**